examples/README.md

```diff
@@ -4,20 +4,10 @@ This folder contains a number of different examples to help you get up and runni
 
 ## Running the Examples
 
-To run the examples, start by installing and building everything from the root directory of the repo. This will install the dependencies and build all of the packages and examples
+From the root of the repo, run the following command to build packages and run an example:
```

Comment:
I wonder if we should still document how to run an example from within the folder. We have the instructions in all the READMEs already so maybe we just have a sentence somewhere that says "You can also build and run an example from it's folder. Check the example's README for details."

Reply:
good idea